desc/plotting.py

```diff
@@ -879,6 +881,8 @@ def plot_fsa(
         Object from which to plot.
     name : str
         Name of variable to plot.
+    axis : bool, optional
```

Comment:
Is this needed? My thinking is that the only reason we would not want to plot the axis is if we expect it to be NaN, but matplotlib will ignore NaNs anyways, so I think the plot would basically look the same whether we include a point at the axis or not.